proto/protovalidate/buf/validate/validate.proto

```diff
@@ -206,6 +206,19 @@ message FieldConstraints {
   }
 }
 
+// Specifies how FieldConstraints.ignore behaves. See the documentation for
+// FieldConstraints.required for definitions of "populated" and "nullable".
+enum Ignore {
+  // Validation is never skipped on the field unless it's an unpopulated
+  // nullable field.
+  IGNORE_UNSPECIFIED = 0;
+  // Validation is skipped if the field is unpopulated.
+  IGNORE_EMPTY = 1;
+  // Validation is skipped if the field is unpopulated or if it is a nullable
+  // field populated with its default value.
+  IGNORE_EMPTY_AND_DEFAULT_VALUE = 2;
```

Comment:
```suggestion
  // Validation is skipped if the field is unpopulated or if it is a nullable
  // field populated with its default value.
  IGNORE_DEFAULT = 2;
```
nit: maybe calling `IGNORE_DEFAULT` is a bit more concise and doesn't loose the meaning?

Reply:
Yep, I think the wording isn't going to matter for the confusion though 😆.

From the URL validation I always wanted the ignore default behaviour so would implement `IGNORE_DEFAULT` on both for consistency (which is valid right?):
- `optional string url = 1 [(buf.validate.field).ignore=IGNORE_DEFAULT];`
- `string url = 1 [(buf.validate.field).ignore=IGNORE_DEFAULT];`
 
We could add some `IGNORE_EMPTY` examples to help clarify use cases?